### PR TITLE
Base setup for app

### PR DESCRIPTION
**Pull request by johnsunam**

### CODE REVIEW

File: comp/app/workstream-page/build/cnm.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      MemcachedEndPoint:
+        Fn::ImportValue: !Join [':', [!Ref EnvironmentName, !Ref UIRSubSystem, MemcachedEndpoint]]
+      MemcachedPort:
+        Fn::ImportValue: !Join [':', [!Ref EnvironmentName, !Ref UIRSubSystem, MemcachedPort]]
+      RedisEndpoint:
+        Fn::ImportValue: !Join [':', [!Ref EnvironmentName, !Ref UIRSubSystem, RedisEndpointAddress]]
+      RedisPort:
+        Fn::ImportValue: !Join [':', [!Ref EnvironmentName, !Ref UIRSubSystem, RedisHostPort]]
+      RedisPassword: '{{resolve:ssm:elasticache-auth-token:1}}'
+      MediaCDNUrl:
```
Comment (rojesh) [inline comp/app/workstream-page/build/cnm.yaml:67]: Not needed.

File: comp/app/workstream-page/build/cnm.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    AllowedPattern: "^[a-z]+$"
+    Default: uir
+
+  CollectorService:
+    Type: String
+    Description: Tracing Collector service name
+    Default: tracing-collector
+
+Resources:
+  PublishToCNM:
```
Comment (rojesh) [inline comp/app/workstream-page/build/cnm.yaml:50]: I don't see config.template.json file. Isn't it needed?

File: comp/app/workstream-page/build/service.yaml [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+      Policies:
+      - PolicyName: !Sub ${AWS::StackName}-service-policy
+        # NOTE: This is dummy policy for the microservice
+        PolicyDocument:
+          Statement:
+          - Effect: Allow
+            Action:
+              - "dynamodb:*"
+            #FIXME: This needs to fine grained rather than blanket *
+            Resource: '*'
```
Comment (milap-neupane) [inline comp/app/workstream-page/build/service.yaml:182]: Same here. Does this need dynamodb access?

Comment (johnsunam) [reply comp/app/workstream-page/build/service.yaml:181]: I have removed it.

File: comp/app/workstream-page/src/express/nodemon.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+    "watch": ["src"],
+    "ext": "ts",
+    "ignore": ["src/**/*.spec.ts"],
+    "exec": "ts-node -r tsconfig-paths/register src/main.ts"
+}
+
```
Comment (rojesh) [inline comp/app/workstream-page/src/express/nodemon.json:7]: eof

File: comp/app/workstream-page/build/service.yaml (hunk 3 above)
Comment (milap-neupane) [reply comp/app/workstream-page/build/service.yaml:181]: @johnsunam Actually this is even worst as we are allowing all the action here for all resource. Does it need to access any AWS resources? I think it does not need any resource access. So it is better to remove the `Policies` section while defining the `AWS::IAM::Role`

Comment (johnsunam) [reply comp/app/workstream-page/build/service.yaml:181]: ok dai.

**Pull request merged**